esmvalcore/preprocessor/_io.py

```diff
@@ -102,7 +113,19 @@ def load(
 
     """
     if isinstance(file, (str, Path)):
-        cubes = _load_from_file(file, ignore_warnings=ignore_warnings)
+        extension = (
+            file.suffix
+            if isinstance(file, Path)
+            else os.path.splitext(file)[1]
+        )
+        if "zarr" not in extension:
+            cubes = _load_from_file(file, ignore_warnings=ignore_warnings)
```

Comment:
`_load_zarr` loads a file, right? In that case, I would put all the logic into `_load_from_file`. In there, we already distinguish between GRIB and netCDF files, so it would be very fitting to also include zarr there.

Reply:
good point! Let me do this, after I popped all the other suggested via GH code changes, will follow up with a commit hash

Reply:
hmmm having second thoughts about this...will write a PR comment below now